ci_framework/roles/libvirt_manager/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure output directory exists
+  ansible.builtin.file:
+    path: "{{ cifmw_libvirt_manager_basedir }}/artifacts"
+    state: directory
+
+- name: Virtualization check and configuration tasks
+  ansible.builtin.include_tasks: virtualization_prerequisites.yml
+
+- name: Install packages and dependencies
+  ansible.builtin.include_tasks: packages.yml
+
+- name: Add polkit rules
+  ansible.builtin.include_tasks: polkit_rules.yml
+
+- name: Check Virsh usage
+  ansible.builtin.include_tasks: virsh_checks.yml
+
+# - name: Deploy EDPM Computes
```

Comment:
Actually, that one would need CRC, since it does some stuff with CRC network. We probably want to keep it "on the side", and included it at the right moment in the main playbook.